include/parser/stream.h

```diff
@@ -125,6 +234,19 @@ class StreamParser {
     return res;
   }
 
+  std::vector<uint8_t> getBytesUntil(const QString &name, uint8_t termination,
+                                     bool include_termination = true) {
+    auto data =
+        getDataUntil(name, data::RepackFormat{data::RepackEndian::LITTLE, 8},
+                     data::BinData::fromRawData(8, {termination}),
+                     data::FieldHighType(), include_termination);
+    std::vector<uint8_t> res;
+    for (size_t i = 0; i < data.size(); i++) {
+      res.push_back(data.element64(i));
```

Comment:
the same code is right now in getBytes, so maybe latter we can separate common code into another function and make some improvments here